tests/qat/test_utilities.py

```diff
@@ -71,6 +73,53 @@ def test_gaussian_function_first_derivative():
     assert y_x[1] == -y_x[-2]
 
 
+@pytest.mark.parametrize(
+    ["width", "std_dev", "zero_at_edges"],
+    [
+        (width, std_dev, zero_at_edges)
+        for width in [0.5, 1.0, 1.5]
+        for std_dev in [0.05, 0.1, 1.0]
+        for zero_at_edges in [0, 1]
+    ],
+)
+def test_gaussian_zero_edge(width, std_dev, zero_at_edges):
+    x = np.linspace(-width / 2, width / 2, 101)
+    gaussian_zero = GaussianZeroEdgeFunction(std_dev, width, zero_at_edges)
+    y = gaussian_zero.eval(x)
+
+    assert np.isclose(max(y), 1.0)
+    assert np.isclose(x[np.argmax(y)], 0.0)
+    if zero_at_edges:
+        assert np.isclose(y[0], 0.0) and np.isclose(y[-1], 0.0)
+    else:
+        assert y[0] > 0.0 and y[-1] > 0.0
+
+
+@pytest.mark.parametrize(
+    ["width", "std_dev", "zero_at_edges"],
+    [
+        (width, std_dev, zero_at_edges)
+        for width in [0.5, 1.0, 1.5]
+        for std_dev in [0.05, 0.1, 1.0]
+        for zero_at_edges in [0, 1]
+    ],
+)
+def test_gaussian_square(width, std_dev, zero_at_edges):
+    x = np.linspace(-2.0, 2.0, 101)
+    gaussian_square = GaussianSquareFunction(width, std_dev, zero_at_edges)
+    y = gaussian_square.eval(x)
```

Comment:
super minor, I think these functions are callable, you could just say `gaussian_square(x)` which calls the `eval()` underneath